py3status/modules/weather_owm.py

```diff
@@ -0,0 +1,758 @@
+        (default '☁')
+    icon_extreme: Icon for extreme weather
+        (default '⚠')
+    icon_humidity: Icon for humidity
+        (default '●')
+    icon_pressure: Icon for pressure
+        (default '◌')
+    icon_rain: Icon for rain
+        (default '🌧')
+    icon_snow: Icon for snow
+        (default '❄')
+    icon_sun: Icon for sunshine
+        (default '☼')
+    icon_sunrise: Icon for sunrise
+        (default '⇑')
+    icon_sunset: Icon for sunset
+        (default '⇓')
+    icon_temperature: Icon for temperature
+        (default '○')
+    icon_thunderstorm: Icon for thunderstorms
+        (default '⛈')
+    icon_wind: Icon for wind or breeze
+        (default '☴')
+    icons: A dictionary relating weather code to icon
+        See https://openweathermap.org/weather-conditions for a complete list
+        of supported icons. This will fall-back to the listed icon if there is
+        no specific icon present.
+        There are multiple ways to specify individual icons based on the id:
+            - Use the key 'i601' to reference the condition with id = 601
+              (snow)
+            - Use the key 'i230_i232' to reference a span of conditions
+              inclusive, in this case conditions (230, 231, 232) (thunderstorm
+              with drizzle)
+        (default None)
+    lang: An ISO 639-1 code for your language (two letters)
+        (default 'en')
+    location: A tuple of floats describing the desired weather location
+        The tuple should follow the form (latitude, longitude), and if set,
+        implicitly disables the Timezone API for determining location.
+        (default None)
+    offset_gmt: A string describing the offset from GMT (UTC)
+        The string should follow the format '+12:34', where the first
+        character is either '+' or '-', followed by the offset in hours,
+        then the offset in minutes. If this is set, it disables the
+        automatic timezone detection from the Timezone API.
+        (default None)
+    rain_unit: Unit for rain fall
+        When specified, a unit may be any combination of upper and lower
+        case, such as 'Ft', and still be considered valid as long as it is in
+        the below options.
+        Options:
+            cm, ft, in, mm, m, yd
+        (default 'in')
+    request_timeout: The timeout in seconds for contacting the API's.
+        (default 10)
+    snow_unit: Unit for snow fall
+        Options:
+            cm, ft, in, mm, m, yd
+        (default 'in')
+    temp_unit: Unit for temperature
+        Options:
+            c, f, k
+        (default 'f')
+    thresholds: Configure temperature colors based on limits
+        The numbers specified inherit the unit of the temperature as configured.
+        The default below is intended for Fahrenheit. If the set value is empty
+        or None, the feature is disabled.
+        (default [(-100, '#0FF'), (0, '#00F'), (50, '#0F0'), (150, '#FF0')])
+    wind_unit: Unit for wind speed
```

Comment:
Good call, it would certainly help with consistency since we have `icon_xxx` and `format_xxx`.

Fixed!